dashboard/public/pixelation/pixelation.html

```diff
@@ -90,8 +90,42 @@
     $('#below_viz_instructions').text(options.instructions).show();
   }
   pixelationInit();
-  pixel_data.value = options.data;
-  drawGraph();
-  if(options.format=='raw') unformatBits()
-  else formatBitDisplay();
+
+  function pixelationDisplay() {
+    pixel_data.value = options.data;
+    drawGraph();
+    formatBitDisplay();
+  }
+
+  // Initialize projects for save/load functionality if channel id is present.
+  if (appOptions.channel) {
+    window.apps.setupProjectsExternal();
+    var projectsHandler = {
+      setInitialLevelHtml: function (levelHtml) {},
+      getLevelHtml: function () {
+        return '';
+      },
+      setInitialLevelSource: function (levelSource) {
+        options.data = levelSource;
```

Comment:
I disagree with Brent here -- I don't think `pixelationOptions` is any better than `options` and makes the PR diff harder to read.  Also we end up in a weird place where we have both `window.options` and `window.pixelationOptions` that contain the same data but aren't `===`, until this TODO is resolved: https://github.com/code-dot-org/code-dot-org/commit/655fe481b0bc65be2b51044c0c11c5f63703688b#diff-5fa6f99eeb1a432ed02c2d5800d32e3fR4 (when?).